csi/src/dev/iscsi.rs

```diff
@@ -116,6 +116,13 @@ impl TryFrom<&Url> for IscsiDevice {
 
 #[tonic::async_trait]
 impl Attach for IscsiAttach {
+    async fn parse_parameters(
```

Comment:
Do we need to set the same timeout for iscsi?

Reply:
ioTimeout setting in storage class is valid only for nvmf protocol. If set for another protocol then create volume request fails. So the answer is that we don't need to set it for iscsi because it's not supported. However that might change in the future if there is a motivation to improve iscsi for whatever reason.

Reply:
Ah ok, is it documented that the ioTimeout is only valid for nvmf?

Reply:
ioTimeout parameter as such isn't documented. It should be documented in our gitbook. good point 👍